release/preview/nanoserver1809/meta.json

```diff
@@ -15,4 +24,4 @@
     "TestProperties": {
         "size": 1
     }
-}
+}
```

Comment:
please make this posix, and add the newline back.

Reply:
```suggestion
}

```

Reply:
I'm not sure what you mean by posix here.